src/pages/CreateJobProfile/CreateJobProfilePage.jsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Typography } from "@mui/material";
+import TextField from "@mui/material/TextField";
+import Button from "@mui/material/Button";
+import PageLayout from "../../components/PageLayout";
+
+/**
+ * Renders the create job profile page.
+ * @returns {JSX.Element} The create job profile page component.
+ */
+function CreateJobProfilePage() {
+
+    return (
+        <PageLayout title="Create Job Profile" viewportPage>
+            <Box
+                sx={{
+                    display: "flex",
+                    flexDirection: "column",
+                    justifyContent: "center",
+                    alignItems: "center",
+                }}
+            >
+                <Typography variant="h3">Insert details of the job</Typography>
+                <TextField
+                    id="jobName"
+                    label="Job Name"
+                    variant="outlined"
+                    size="medium"
+                    sx={{ mt: 4, width: "60%" }}  
+                />
+                <TextField
+                    id="jobDescription"
+                    label="Job Description"
+                    variant="outlined"
+                    size="medium"
+                    multiline
+                    rows={15}
+                    sx={{ mt: 4, width: "60%" }} 
```

Comment:
I struggle with this width: 60%. On smaller screens you waste a lot of space. Maybe some breakpoints?

Reply:
image.png
I tried it with the smallest screen and I think it is okay

Reply:
Your image is not loading

Reply:
<img width="568" alt="Bildschirmfoto 2023-11-16 um 09 41 36" src="https://github.com/SE2-WS23/Skillverse-Web/assets/116264057/13c0c1f2-c835-4ac1-b2ed-204a6b14a2af">

Reply:
I still think this is a waste of space, but @NicolasKmbr should decide this

Reply:
I think 60% is good